augmentation/augmentation.hpp

```diff
@@ -176,7 +176,7 @@ class Augmentation
   //! Locally held value of augmentation probability.
   double augmentationProbability;
 
-  // The dataloader class should have access to internal functions of
+  // The augmentation class should have access to internal functions of
```

Comment:
Since we use functions from augmentation class by accessing them through dataloader class. So maybe we can word it as the dataloader class should have access to internal functions of augmentation class.

Reply:
Ohh..my bad. Fixed it :)